qt/widgets/common/src/DataProcessorUI/AbstractDataProcessorTreeModel.cpp

```diff
@@ -1,28 +1,29 @@
-#include "MantidQtWidgets/Common/DataProcessorUI/AbstractDataProcessorTreeModel.h"
```

Comment:
It looks like this file is a duplicate of AbstractTreeModel.cpp that's been left in the repo by mistake?